src/lib/services/feature-toggle-service.ts

```diff
@@ -1967,7 +1968,26 @@ class FeatureToggleService {
     }
 
     async updatePotentiallyStaleFeatures(): Promise<void> {
-        await this.featureToggleStore.updatePotentiallyStaleFeatures();
+        const potentiallyStaleFeatures =
+            await this.featureToggleStore.updatePotentiallyStaleFeatures();
+        if (this.flagResolver.isEnabled('emitPotentiallyStaleEvents')) {
+            if (potentiallyStaleFeatures.length) {
```

Comment:
Should you compare length to 0?

Reply:
Sure, we can do that! Probably more explicit, so I think that's a sensible change.

Reply:
cf2cf015ced7d37548c6c7c3e1829ba4ca390550